lib/Parser/Parse.cpp

```diff
@@ -5040,6 +5040,17 @@ ParseNode * Parser::ParseFncDecl(ushort flags, LPCOLESTR pNameHint, const bool n
         }
     }
 
+    if (fDeclaration)
+    {
+        Symbol* funcSym = pnodeFnc->GetFuncSymbol();
+        if (funcSym != nullptr &&
```

Comment:
(Note that for a declaration the symbol must be non-null.)

Reply:
In case of deferred parsing (-force:deferparse) ParseFncNames doesn't assign to pnodeFnc->pnodeName so symbol is null. Need to find a way to fix that.